content/en/docs/ops/common-problems/upgrade-issues/index.md

```diff
@@ -0,0 +1,149 @@
+---
+title: Upgrade Problems
+description: Resolve common problems with Istio upgrades.
+weight: 40
+owner: istio/wg-policies-and-telemetry-maintainers
+test: n/a
+---
+
+## `EnvoyFilter` migration
```

Comment:
In the generated page this EnvoyFilter in the title is not look a bit odd, because of the back ticks. https://deploy-preview-13360--preliminary-istio.netlify.app/latest/docs/ops/common-problems/upgrade-issues/. Same issue with L72. May be it makes sense to remove the back ticks, and add these words to .spelling?

Reply:
Removed backticks.